main.py

```diff
@@ -3,7 +3,9 @@
 from flask_cors import cross_origin
 from flask_talisman import Talisman
 from flask_swagger_ui import get_swaggerui_blueprint
-from utils import perform_notebook_conversion, perform_rmd_conversion, authorized
+import convert_rmd as rmd_converter
```

Comment:
Why the renaming? I would suggest to either rename the files, or stick with the original name to avoid confusion

Reply:
I can rename the files. "convert_rmd" collided with the endpoint name, but I wanted to retain the pattern of naming the helper modules as verbs.